content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -197,6 +197,8 @@ service PodResourcesLister {
 }
 ```
 
+### `List` gRPC endpoint
```

Comment:
I'd set the fragment identifier explicitly:
```suggestion
### `List` gRPC endpoint {#grpc-endpoint-list}
```

Reply:
Ack.